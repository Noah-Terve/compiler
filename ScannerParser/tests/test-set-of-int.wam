set<int> exampleSet= {4, 5, 6, 1};