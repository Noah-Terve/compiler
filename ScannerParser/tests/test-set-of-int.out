Set <int> exampleSet = {4, 5, 6, 1};
